lib/Chrome/ChromeDriverService.php

```diff
@@ -19,19 +19,23 @@
 
 class ChromeDriverService extends DriverService
 {
-    // The environment variable storing the path to the chrome driver executable.
+    /**
+     * The environment variable storing the path to the chrome driver executable.
+     * @deprecated Use ChromeDriverService::CHROME_DRIVER_EXECUTABLE
+     */
     const CHROME_DRIVER_EXE_PROPERTY = 'webdriver.chrome.driver';
+    // The environment variable storing the path to the chrome driver executable
+    const CHROME_DRIVER_EXECUTABLE = 'WEBDRIVER_CHROME_DRIVER';
 
     /**
      * @return static
      */
     public static function createDefaultService()
     {
-        $exe = getenv(self::CHROME_DRIVER_EXE_PROPERTY);
+        $exe = getenv(self::CHROME_DRIVER_EXE_PROPERTY) ?: getenv(self::CHROME_DRIVER_EXECUTABLE);
```

Comment:
This will give priority to the old value.
Given the old version is deprecated, shouldn't this be swapped, i.e.

```
    $exe = getenv(self::CHROME_DRIVER_EXECUTABLE) ?: getenv(self::CHROME_DRIVER_EXE_PROPERTY);
```